tests/Js/WindowTest.php

```diff
@@ -82,6 +82,34 @@ public function testResizeWindow()
         $this->assertTrue($session->evaluateScript($jsWindowSizeScript));
     }
 
+    public function testResizeWindowBeforePageVisit()
+    {
+        $session = $this->getSession();
+        $session->resizeWindow(400, 300);
+        $session->wait(1000, 'false');
```

Comment:
using the session to wait without a meaningful JS condition (i.e. one subject to change) is useless. Just use ``sleep``.